src/Bridges/Variable/bridges/rsoc_soc.jl

```diff
@@ -93,3 +93,11 @@ function MOI.Bridges.inverse_adjoint_map_function(
 )
     return MOI.Bridges.map_function(BT, func)
 end
+
+function MOI.supports(
+    ::MOI.ModelLike,
+    ::MOI.VariablePrimalStart,
+    ::Type{<:RSOCtoSOCBridge},
+)
+    return false
+end
```

Comment:
I've opted-out here, since setting or getting the start can't be done in isolation.

Reply:
Yes, this is part of the optional API. By the way, this should be default as in
https://github.com/jump-dev/MathOptInterface.jl/blob/5025b50de1d6cb29e6bbb0d26adfcf1b57b6a51c/src/Bridges/bridge.jl#L105-L149

Reply:
Bridges should have to implement it to return `false`